conans/test/unittests/util/tools_test.py

```diff
@@ -1220,6 +1225,22 @@ def get_values(this_os, this_arch, setting_os, setting_arch, compiler=None):
         self.assertEquals(build, "x86_64-apple-darwin")
         self.assertEquals(host, "aarch64-apple-darwin")
 
+        build, host = get_values("Darwin", "x86_64", "tvOS", "armv8.3")
+        self.assertEquals(build, "x86_64-apple-darwin")
+        self.assertEquals(host, "aarch64-apple-darwin")
```

Comment:
should this result be different from the triplet for `armv8` ?? Right now it is the same, but not sure how apple tools will differentiate it from the other

Reply:
Finally @SSE4 agreed on leaving `aarch64` for 64-bit ARMs, so I think it is ok as is.